### PR TITLE
Import fail if Python is running with -OO

### PR DESCRIPTION
**Pull request by JGoutin**
Hello,

This fix a crash on Astropy import if Python compilation optimization is set to 2 (Remove docstrings / Run python with -OO argument)

### CODE REVIEW

**Comment by embray** [comment]
Thanks for this!  Incidentally, I fixed a bunch of these back in  #3923, but looks like somehow I missed this one.  Curious that it didn't show up in my testing, but oh well.

**Comment by embray** [comment]
Can you please add a changelog entry, as a bug fix to `io.votable` in 1.0.6?

**Comment by JGoutin** [comment]
Added to changelog.

**Comment by mhvk** [comment]
The documentation build failed on this one, perhaps because of a missing newline or so (it worries about indendation level). Maybe check an example from another bug fix entry...

File: CHANGES.rst [hunk 1]
```diff
@@ -843,6 +843,8 @@ Bug Fixes
 - ``astropy.io.registry``
 
 - ``astropy.io.votable``
```
Comment (embray) [inline CHANGES.rst]: Indeed as @mhvk noted there should be a newline here.  I'll rebase and fix it.

**Pull request closed without merging**